src/tests/FunctionalTests/iOS/Simulator/CoreCLR.Interpreter/iOS.Simulator.CoreCLR.Interpreter.Test.csproj

```diff
@@ -13,8 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <EnvironmentVariables Include="DOTNET_Interpreter=RunInterpreter" />
-    <EnvironmentVariables Include="DOTNET_InterpDump=RunInterpreter" />
+    <EnvironmentVariables Include="DOTNET_InterpDump=*!*" />
     <EnvironmentVariables Include="DOTNET_ReadyToRun=0" />
```

Comment:
Should we be auto-disabling R2R on iOS for now since we're automatically force-enabling the interp instead of using the InterpMode env var? Or alternately, should we use InterpMode instead of hard-coding it in eeinterp.cpp?

Reply:
Reverted to `InterpMode` as it controls tiered compilation as well.